api.ts

```diff
@@ -61,11 +62,25 @@ export class UserVisibleError extends Error {
   }
 }
 
+interface StatusCodeErrorResponse {
+  body?: any;
+  headers?: {[key: string]: string | string[] | undefined};
+}
+
+/**
+ * StatusCodeError is a simple version of StatusCodeError in request-promise to keep backwards compatibility.
```

Comment:
This class is exported in index.ts so it'll show up in public docs. Is this comment intended for Codans or users? If the latter, what does it mean to a user to keep backwards compatibility? I don't think they have context for what this is about.

Reply:
ah good call. let me just remove the comment.